installer/Migration/V4_3_3/Migration.php

```diff
@@ -29,30 +31,118 @@ class Migration extends AbstractMigration
      */
     public function up(): void
     {
-        $this->getSchemaHelper()->changeColumn('hs_hr_employee', 'emp_number', [
-            'Type' => Types::INTEGER, 'Length' => 7, 'Notnull' => true, 'Autoincrement' => true
-        ]);
+        /*$this->getSchemaHelper()->changeColumn('hs_hr_employee', 'emp_number', [
```

Comment:
@RajithaKumara  here we have a small issue adding auto Increment to an existing column
Shall I make a function to update those existing data?

Reply:
Solved by using raw sql